apps/docs/src/app/platform/component-docs/platform-file-uploader/platform-file-uploader-docs.component.ts

```diff
@@ -34,7 +36,8 @@ export class PlatformFileUploaderDocsComponent {
         {
             language: 'html',
             code: platformFileUploadCompactTypesSrc,
-            fileName: 'platform-file-uploader-compact-example'
+            fileName: 'platform-file-uploader-compact-example',
+            scssFileCode: platformFileUploadDefaultTypesScssSrc
```

Comment:
like I said before, different css files are needed for stackblitz to work properly, even if it seems inefficient